client/src/NoteEdit.js

```diff
@@ -1,19 +1,60 @@
-class NoteEdit extends React.Component {
+import React, { Component } from 'react';
+import $ from 'jquery';
+import debounce from 'throttle-debounce/debounce';
+import bootbox from 'bootbox';
+import './keyboard';
+import NoteList from './NoteList';
+import NoteForm from './NoteForm';
+import Note from './Note';
+import PushState from  './PushState';
+import Navbar from './Navbar';
+import SaveStateLabel from './SaveStateLabel';
+import AddNoteButton from './AddNoteButton';
+import AlertFlash from './AlertFlash';
+import EventHive from './EventHive';
+import AutoSave from './AutoSave';
+
+class NoteEdit extends Component {
   constructor(props, context) {
     super(props, context);
 
+    this.pushState = new PushState(this.props.history);
+
+    const { match } = this.props;
+
     if (props.note) {
       this.state = {
         isInitialEdit: true,
         note: Note.fromAttributes(props.note)
       };
-      PushState.setBrowserTitle(props.note);
+      this.pushState.setBrowserTitle(props.note);
```

Comment:
you can also use https://github.com/nfl/react-helmet to set browser title (or any other attribute in `<head>`. Cool thing about helmet is, that it also works with server-side-rendering